server/src/main/java/com/cloud/network/element/VirtualRouterElement.java

```diff
@@ -272,12 +272,15 @@ public boolean prepare(final Network network, final NicProfile nic, final Virtua
             return false;
         }
 
+        Map<VirtualMachineProfile.Param, Object> params = new HashMap<>();
```

Comment:
```suggestion
        implement(network, offering, dest, context);
```

Reply:
@DaanHoogland 
right. all lines before `return true`

github did not work well when I selected many lines. .

Reply:
i know, to bad, it would help with suggestions like these.